### PR TITLE
2.3.1

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
There was a patch. Here's an update.

### CODE REVIEW

File: com.shatteredpixel.shatteredpixeldungeon.yaml [hunk 1]
```diff
@@ -42,8 +42,8 @@ modules:
 
   sources:
   - type: file
-    url: https://github.com/00-Evan/shattered-pixel-dungeon/releases/download/v2.3.0/ShatteredPD-v2.3.0-Java.jar
-    sha256: 6f60c3198aca52f4946e8629ec1ea1a69fd4661f1823475f6af69113681cd261
+    url: https://github.com/00-Evan/shattered-pixel-dungeon/releases/download/v2.3.1/ShatteredPD-v2.3.0-Java.jar
```
Comment (TheShadowOfHassen) [inline com.shatteredpixel.shatteredpixeldungeon.yaml:45]:
```suggestion
    url: https://github.com/00-Evan/shattered-pixel-dungeon/releases/download/v2.3.1/ShatteredPD-v2.3.1-Java.jar
```

**Comment by TheShadowOfHassen** [comment]
That should fix it.

**Comment by TheShadowOfHassen** [comment]
It works!!!

**Comment by dschier-wtd** [comment]
looking good. Thank you for the continuous contribution,

Next time, it would be awesome, if you can update the screenshots, too. That makes it easier for users to discover new features.

**Comment by TheShadowOfHassen** [comment]
Evan said he'd probably do another bug fix before 2.4, so I can add better screenshots then.

Also, does the shattered pixel dungeon flatpak need any updates so it falls under the new requirements?

**Comment by dschier-wtd** [comment]
Afaik, we are pretty good. Links, Icon and sorts are maintained properly. Permissions are also limited. :)

**Pull request merged**